test/autorest/generated/lrogroup/pollers.go

```diff
@@ -31,7 +31,7 @@ type lrOSCustomHeaderPost202Retry200Poller struct {
 
 // Poll returns false if there was an error or polling has reached a terminal state
 func (p *lrOSCustomHeaderPost202Retry200Poller) Poll(ctx context.Context) bool {
-	return !p.pt.done(ctx, p.client.p)
+	return !done(ctx, p.client.p, p.pt)
```

Comment:
Since this is no longer a method on the tracker we might want to give this a less generic name, maybe `lroDone` or `lroTerminated` or something.  It's internal so the name doesn't really matter, but might be nice.